### DISCUSSION TITLE
Is there a way to archive an app in envkey?

### DISCUSSION THREAD

**Discussion by gee-forr** [envkey/envkey#12] [Q&A] [answered]
Hi there,

We have a lot of apps in our Envkey instance, and some of them we don't work on for a very long time. This means that we cannot delete them, as we'll need to come back to them at some point down the road, but in the meantime, they clutter up the app list.

Is there a way to archive an app without deleting it? Or is there some other way to declutter a large app list with dormant apps? 

Perhaps a way to group apps and collapse groups we don't frequently use (and obv. Envkey would need to remember that we've collapsed the group too).

Let me know.

**Comment by danenania** [comment, marked as answer]
There's not currently a way to do this, but some groundwork has already been laid out for app groups (and block groups). App groups + sticky collapse/expand in the UI menu sounds like it could be a good way to deal with this.

**Reply by gee-forr** [reply]
Fantastic - looking forward to seeing those changes one day :)
